resources/control_systems.md

```diff
@@ -5,66 +5,712 @@
+
+For an input signal $r(t) = A\sin(\omega_0 t)$ and open loop transfer function $G(s) = G(j\omega)$, with
+magnitude and phase $G(j\omega_0)|\angle G(j\omega_0)|$
+the output signal is
+
+$$c(t) = A|G(j\omega_0)|\sin(\omega_0 t+\angle G(j\omega_0))$$
+
+- A is the amplitude of the input signal
+- $\omega_0$ is the angular frequency of the input signal, mathematically described as
+
+$$\omega_0 = 2\pi f_0$$
+
+- $f_0$ is the frequency of the input signal
+
+## 13 Frequency Domain Specifications
+
+For the 2nd order transfer function
+
+$$T(S) = \frac{C(s)}{R(s)} = (\frac{w_n^2}{s^2+2\zeta w_n s+
+w_n^2})$$
+
+The magnitude of _T(s)_, with $s = j\omega$,
+
+$$M = |T(j\omega)| = \frac{1}{\sqrt{(1-(\frac{\omega}{\omega_n})^2)
+^2+(2\zeta \frac{\omega}{\omega_n})^2}}$$
+
+The phase of _T(s)_, with $s = j\omega$,
+
+$$\angle T(j\omega) = -\tan^{-1}(\frac{\frac{2\zeta\omega}{\omega_n}}
+{1 - (\frac{\omega}{\omega_n})^2})$$
+
+### 13.1 Resonant Frequency
+
+Resonant frequency, $\omega_r$, is the frequency when the magnitude of the frequency response
+reaches its peak value for the first time.
+
+It is mathematically defined as
+
+$$\omega_r = \omega_n\sqrt{1-2\zeta^2}$$
+
+### 13.2 Peak Resonance
+
+Peak resonance, $M_r$ is the maximum magnitude of $T(j\omega$).
+
+It is mathematically defined as
+
+$$M_r = \frac{1}{2\zeta\sqrt{1-\zeta^2}}$$
+
+### 13.3 Bandwidth
+
+Bandwidth, $\omega_b$, is the frequency range over which, the magnitude of $T(j\omega$) drops to 70.7%
+of its zero frequency value.
+
+$$\omega_b = \omega_n\sqrt{1-2\zeta^2+
+\sqrt{2-4\zeta^2+4\zeta^4}}$$
+
+## 14 Control System Plots
+
+### 14.1 Bode Plot
+
+Bode Plots describe the change in magnitude and phase as a function of frequency.
+
+The magnitude of an open loop transfer function in dB is
+
+$$M = 20\log(|G(j\omega)H(j\omega)|)$$
+
+The phase angle of an open loop transfer function in degrees is
+
+$$\phi = \angle G(j\omega)H(j\omega)$$
+
+**Recommended video:**
+> [Introduction to Bode Plots](https://www.youtube.com/watch?reload=9&v=gZhRvrGSZdQ)
+
+**Recommended video:**
+> [Bode Plot Solved Example](https://www.youtube.com/watch?v=JLWJTdK0Q0w)
+
+#### 14.1.1 Stability Analysis with Bode Plots
+
+##### 14.1.1.1 Cross Over Frequency
+
+- **Phase Cross Over Frequency**, $\omega_pc$: This is the frequency at which the bode phase plot has a phase of
+  $-180^\circ$. Its unit is **rad/sec**.
+- **Gain Cross Over Frequency**, $\omega_gc$: This is the frequency at which the bode magnitude
+  plot has a magnitude of zero dB. Its unit is **rad/sec**.
+
+| Stability Criteria      | Stability                |
+|-------------------------|--------------------------|
+| $\omega_pc > \omega_gc$ | Stable system            |
+| $\omega_pc = \omega_gc$ | Marginally Stable system |
+| $\omega_pc < \omega_gc$ | Unstable system          |
+
+##### 14.1.1.2 Gain and Phase Margin
+
+-**Gain Margin**, GM: This is the negative value of magnitude in dB at the phase cross over frequency, $M_pc$.
```

Comment:
Minor mistake on point form `-Some Text` resulting in no proper markdown rendering.